server/events_controller.go

```diff
@@ -146,6 +146,7 @@ func (e *EventsController) handleGithubPost(w http.ResponseWriter, r *http.Reque
 	e.Logger.Debug("request valid")
```

Comment:
Can we put the log into this existing "request valid" logs for every place that has it? And then we don't need the additional "handling" log.

```suggestion
	githubReqID := "X-Github-Delivery=" + r.Header.Get("X-Github-Delivery")
	e.Logger.Debug("request valid %s",  githubReqID)
```